glusterd2/xlator/load.go

```diff
@@ -148,10 +147,10 @@ func getXlatorsDir() string {
 func loadAllXlators() (map[string]*Xlator, error) {
 
 	xlatorsDir := getXlatorsDir()
-
 	if xlatorsDir == "" {
 		return nil, fmt.Errorf("No xlators dir found")
 	}
+	log.WithField("xlatordir", xlatorsDir).Debug("Xlators dir found")
```

Comment:
If you need to handle #666 then you should check for version < 4.0.0 here?

Reply:
That's been fixed in https://github.com/gluster/glusterd2/pull/667